src/button-layout.tsx

```diff
@@ -204,18 +139,52 @@ const ButtonLayout: React.FC<ButtonLayoutProps> = ({
 
     const needsLinkAlignment = !isTabletOrSmaller && align === 'left';
 
+    const getJustifyContent = () => {
+        if (align === 'center' || (align === 'full-width' && isTabletOrSmaller)) {
```

Comment:
Can why try to use media queries instead of `useScreenSize`?

By using media queries we avoid hydration issues (`useScreenSize` does not work in SSR) or style flashes (first render to match server, second client render with the correct mq values).

Now that we're refactoring the styles, it's a good moment to rethink this kind of things